runtime/autoload/man.vim

```diff
@@ -310,20 +308,37 @@ function! s:format_candidate(path, psect) abort
 endfunction
 
 function! man#init_pager() abort
-  " Remove all backspaced characters.
-  execute 'silent keeppatterns keepjumps %substitute,.\b,,e'.(&gdefault?'':'g')
   if getline(1) =~# '^\s*$'
     silent keepjumps 1delete _
   else
     keepjumps 1
   endif
   " This is not perfect. See `man glDrawArraysInstanced`. Since the title is
   " all caps it is impossible to tell what the original capitilization was.
-  let ref = substitute(matchstr(getline(1), '^[^)]\+)'), ' ', '_', 'g')
+  " But it's usually lowercase, so we'll stick to that.
+  let ref = tolower(substitute(matchstr(getline(1), '^[^)]\+)'), ' ', '_', 'g'))
   try
     let b:man_sect = man#extract_sect_and_name_ref(ref)[0]
   catch
     let b:man_sect = ''
   endtry
   execute 'silent file man://'.fnameescape(ref)
 endfunction
+
+function! man#init_highlight_groups() abort
+  let group = 'Keyword'
+  while 1
+    let values = s:get_hl(group)
+    if values !~# '^links to'
+      break
+    endif
+    let group = matchstr(values, '\w\+$')
+  endwhile
+  execute 'highlight default manBold' values 'cterm=bold gui=bold'
+  highlight default manUnderline cterm=underline gui=underline
+endfunction
+
+function! s:get_hl(group) abort
+  redir => hl | execute 'silent highlight' a:group | redir END
```

Comment:
can use `execute()` here. Crusty old sneak.vim doesn't because it supports Vim 7.2 :)